Jenkinsfile

```diff
@@ -15,10 +15,10 @@ pipeline {
  			 mkdir build
 			 cd build
 			 cmake -DCMAKE_INSTALL_PREFIX=../install -DSLEEF_SHOW_CONFIG=1 -DENFORCE_TESTER3=TRUE -DBUILD_QUAD=TRUE ..
-			 make -j 6 all
+			 make -j 1 all
```

Comment:
Because add_custom_command does not support parallel builds. In this patch, many COMMANDS are executed within one add_custom_command, and parallel builds fail even on linux computers.

https://stackoverflow.com/questions/60105230/how-to-run-cmake-commands-in-add-custom-command-in-order